dist/RadarSDK.xcframework/Info.plist

```diff
@@ -5,6 +5,8 @@
 	<key>AvailableLibraries</key>
 	<array>
 		<dict>
+			<key>DebugSymbolsPath</key>
```

Comment:
@jrtibbetts How will the addition of dSYM impact the SDK?  I guess we haven' dogfooded it all, eg, in a Toolkit build yet.  cc @jsani-radar

Reply:
It gets stripped out when an app is built, so it shouldn't have any effect. It's just there so that an app developer can upload the debugging symbols to Crashlytics (or another analyzer) separately in order to debug crashes.

Reply:
But I can put it in a Toolkit build first, if you want.